composer.json

```diff
@@ -70,32 +71,38 @@
         ],
         "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests",
         "ci:php:stan": "phpstan --no-progress --configuration=config/phpstan.neon",
+        "ci:php:rector": "\"./vendor/bin/rector\" --no-progress-bar --no-diffs --dry-run --config=config/rector.php",
         "ci:static": [
             "@ci:php:fixer",
-            "@ci:php:stan"
+            "@ci:php:stan",
+            "@ci:php:rector"
         ],
         "ci:tests": [
             "@ci:tests:unit"
         ],
         "ci:tests:sof": "\"./vendor/bin/phpunit\" --stop-on-failure --do-not-cache-result",
         "ci:tests:unit": "\"./vendor/bin/phpunit\" --do-not-cache-result",
         "fix:php": [
-            "@fix:php:fixer"
+            "@fix:php:fixer",
+            "@ci"
```

Comment:
This probably should be `"@fix:php:rector"`, I'd guess?

Reply:
Yes, you're right.